### DISCUSSION TITLE
Bulk Change for Programs

### DISCUSSION THREAD

**Discussion by scharfhunter** [MCLD/greatreadingadventure#754] [Help!]
Hi! Our library system partnered with our local school district this past year to give them access to our GRA site so that they could use it for the 10k students as a reading log and supplemental program. We would like to be able to delete the program so that the summer registrants aren't confused about which program they should be in, and we cannot because 10k profiles are in that school system program. Is there a bulk change option available that I just haven't been able to locate? We'd like to move those profiles out of the school year program so that we can simplify the dropdown menu options for the program possibilities in the profile section, but changing each individual profile's program affiliation will take too long with 10k. Do you have any advice?

**Comment by k7hpn** [comment]
The software doesn't currently have an interface in it to do a "clean start" for a new program. The most straightforward approach is to point the software at an empty database - from that point you can go through the initial configuration steps again - this is how we start our summer reading process every year. You may want to ensure you run reports on relevant data or save a backup of the database before you delete it (or just leave it on the database server and create a new database to point the software at).

I hope that helps, let me know if it does or it doesn't!
